meters/metric.rb

```diff
@@ -0,0 +1,13 @@
+module Measurement
+  class Metric
+    attr_reader :name, :point_tags
+
+    # @param name [String] metric name
+    # @param point_tags [Hash] metric point tags
+    def initialize(name, point_tags)
+      # TODO: sanitize/validate both?
+      @name = name.to_sym
+      @point_tags = point_tags&.clone || {}
```

Comment:
Now that we are also using `point_tags` for keys, use `.freeze` so that point_tags and hash key are always consistent. I don't think this'll have an issue with `sender.send_metric()` but maybe test with freeze just in case.